src/transformers/models/jukebox/configuration_jukebox.py

```diff
@@ -114,17 +114,30 @@
 _FullDenseAttention = ["dense_attention"]
 _PrimePrimeDenseAttention = ["prime_attn", "prime_attn", "dense_attn"]
 
+
+def FullDenseAttention(layer):
+    return _FullDenseAttention[0]
+
+
+def RawColumnPreviousRowAttention(layer):
+    return _RawColumnPreviousRowAttention[layer % 3]
```

Comment:
Not a fan of functions that have class names, let's lowercase and put underscores ;-)